LBD2024G06TP1Script.sql

```diff
@@ -56,45 +58,67 @@ CREATE TABLE IF NOT EXISTS `LBD2024G06AGROSA`.`EMPLEADOS` (
   `cuil` INT(11) UNSIGNED NOT NULL,
   `nombres` VARCHAR(30) NOT NULL,
   `apellidos` VARCHAR(30) NOT NULL,
-  `estado` CHAR(1) NOT NULL,
+  `estado` CHAR(1) NOT NULL DEFAULT 'A',
   PRIMARY KEY (`idEMPLEADO`),
   UNIQUE INDEX `idEMPLEADOS_UNIQUE` (`idEMPLEADO` ASC) VISIBLE)
 ENGINE = InnoDB
 DEFAULT CHARACTER SET = utf8;
 
+-- Added table DUEÑOS
+
+CREATE TABLE IF NOT EXISTS `LBD2024G06AGROSA`.`DUEÑOS` (
+  `cuil` INT(11) NOT NULL,
+  `nombres` VARCHAR(30) NOT NULL,
+  `apellidos` VARCHAR(30) NOT NULL,
+  `estado` CHAR(1) NOT NULL DEFAULT 'A',
+  PRIMARY KEY (`cuil`))
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+-- Updated table FINCAS
+
 CREATE TABLE IF NOT EXISTS `LBD2024G06AGROSA`.`FINCAS` (
   `idFINCA` INT(11) NOT NULL,
-  `finca` VARCHAR(45) NOT NULL,
-  `responsable` VARCHAR(45) NOT NULL,
-  `cuilResoinsable` INT(11) NOT NULL,
+  `nombreFinca` VARCHAR(45) NOT NULL,
   `latitud` FLOAT(11) NULL DEFAULT NULL,
   `longitud` FLOAT(11) NULL DEFAULT NULL,
-  PRIMARY KEY (`idFINCA`),
+  `cuilDUEÑO` INT(11) NOT NULL,
+  PRIMARY KEY (`idFINCA`, `cuilDUEÑO`),
```

Comment:
It doesnt make sense to make a compound primary key over here. If we already have a unique field that is the ID...